apps/backoffice-v2/src/lib/react-query/mutations/useAssignWorkflowMutation/useAssignWorkflowMutation.tsx

```diff
@@ -1,23 +1,66 @@
 import { useMutation, useQueryClient } from '@tanstack/react-query';
 import { api } from '../../../../api/api';
-import { Action, Resource } from '../../../../enums';
+import toast from 'react-hot-toast';
+import { t } from 'i18next';
+import { useUsersQuery } from '../../queries/useUsersQuery/useUsersQuery';
 
 export const useAssignWorkflowMutation = ({ workflowRuntimeId }: { workflowRuntimeId: string }) => {
   const queryClient = useQueryClient();
+  const { data: users } = useUsersQuery();
+
   return useMutation({
-    mutationFn: ({ assigneeId, isAssignedToMe }: { assigneeId: string; isAssignedToMe: boolean }) =>
+    mutationFn: ({ assigneeId }: { assigneeId: string | null; isAssignedToMe: boolean }) =>
       api.workflows.updateById({
         workflowId: workflowRuntimeId,
         body: {
           assigneeId,
         },
       }),
-    onMutate: ({ isAssignedToMe }) => ({
-      resource: Resource.ASSIGNMENT,
-      action: isAssignedToMe ? Action.ASSIGNED_TO_ME : Action.ASSIGNED_TO_OTHER,
-    }),
-    onSuccess: () => {
+    onMutate: ({ assigneeId }) => {
+      const assigneeName = assigneeId
```

Comment:
The assigneeId check can be omitted. Just make sure that the line is updated to include optional chaining before fullName, i.e. `?.fullName`.